__tests__/app.test.js

```diff
@@ -0,0 +1,26 @@
+const request = require('supertest');
+const data = require('../db/data/test-data/index.js');
+const seed = require('../db/seeds/seed.js');
+const db = require('../db/connection.js');
+const app = require('../app.js');
+
+
+beforeEach(() => seed(data));
+afterAll(() => db.end())
+
+describe('/api/topics', () => {
+    test('GET: 200 returns an array of all objects', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((response) => {
+            const {rows} = response.body
+            rows.forEach(row => {
+                expect(row).toMatchObject({
+                    description: expect.any(String),
+                    slug: expect.any(String),
+                })
+            })
+        })
+    });
+})
```

Comment:
As we have added handling for a 404 error, we want to ideally add a test for this 404. Please resubmit this once you have added the relevant test. 

You may also look to add a 500 test at this point (but this is optional)